client/starwhale/core/model/view.py

```diff
@@ -145,6 +148,19 @@ def tag(self, tags: t.List[str], remove: bool = False, quiet: bool = False) -> N
             console.print(f":surfer: add tags [red]{tags}[/] @ {self.uri}...")
             self.model.add_tags(tags, quiet)
 
+    @classmethod
+    def serve(
```

Comment:
add `@BaseTermView._only_standalone` for serve function?

Reply:
Done